2025.1 Release Notes (#152)
bodo-ai/Bodo: docs/docs/release_notes/2025.01.md

> # Bodo 2025.1 Release (Date: 13/01/2025) {#January_2025}

Reviewer: Use MM/DD/YYYY format.
```suggestion
# Bodo 2025.1 Release (Date: 01/13/2025) {#January_2025}
```

=====================================

### New Features

>  - S3 tables iceberg catalog support for Bodo and BodoSQL.

Reviewer: ```suggestion
 - Bodo now supports S3 Tables (Iceberg) in both Python and SQL.
```
